db/seeds.rb

```diff
@@ -1,7 +1,173 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+puts "Cleaning Database 🧼"
+sleep 1
+print " ."
+sleep 1
+print " ."
+sleep 1
+puts " ."
+sleep 1
+puts ""
+
+# Clear database to prevent duplicate records
+Friendship.destroy_all
+User.destroy_all
+
+# Create users
+emma = User.create(
+  username: "emmvs", 
+  email: "emma@test.com", 
+  password: "123456", 
+  first_name: "Emma", 
+  middle_name: "Anna Agneta", 
+  last_name: "Rünzel",
+  nickname: "Emmsi",
+  emoji: "☀️"
+)
+
+josh = User.create(
+  username: "xxjmsxx", 
+  email: "josh@test.com", 
+  password: "123456", 
+  first_name: "Joshua", 
+  middle_name: "Matthew", 
+  last_name: "Smith",
+  nickname: "Josh",
+  emoji: "🍰"
+)
+
+santi = User.create(
+  username: "santiagosan93", 
+  email: "santi@test.com", 
+  password: "123456", 
+  first_name: "Santiago", 
+  middle_name: nil, 
+  last_name: "Sanchez",
+  nickname: "Santi",
+  emoji: "🐘"
+)
+
+puts "Created #{User.count} Users 🤷🏼‍♂️🧑🏽‍🦱💁🏼‍♀️"
+users = User.find_each
+
+# Create contacts
+users.each do |user|
+  2.times do
+    Contact.create!(
+      name: Faker::Name.name,
+      relationship: ['friend', 'family', 'colleague'].sample,
+      email: Faker::Internet.email,
+      phone_number: Faker::PhoneNumber.cell_phone,
+      address: Faker::Address.full_address,
+      social_media_handle: "@#{Faker::Internet.username}",
+      user: user
+    )
+  end
+end
+
+Contact.create!(
+  name: 'Cindy',
+  relationship: 'friend',
+  email: 'cindyadjei9@gmail.com',
+  phone_number: '+49 176 62423222',
+  address: 'Wutzkyallee 85, 12353 Berlin',
+  social_media_handle: 'afia_agyeman97',
+  user: emma
+)
+
+Contact.create!(
+  name: 'Mamma',
+  relationship: 'family',
+  email: 'sylvia@ruenzel.de',
+  phone_number: '+49 175 9633288',
+  address: 'Buchenweg 97, 22926 Ahrensburg',
+  social_media_handle: 'kramtomat',
+  user: emma
+)
+
+puts "Created #{Contact.count} Contacts 📇"
+
+# Create friendships
+Friendship.create!(
+    status: Friendship.statuses[:requested],
+    requested_at: Time.now,
```

Comment:
Let's get in the habbit of saving all our dates in UTC. This way if we get super viral with this app and we have people all over the world using it, would make things way easiear for us to display dates in multiple time_zones. 
```suggestion
    requested_at: Time.current,
```

Reply:
Yalla 🫡